base/system/smss/sminit.c

```diff
@@ -1922,7 +1945,9 @@ SmpCreateDynamicEnvironmentVariables(VOID)
                                  sizeof(ValueBuffer),
                                  &ResultLength);
         NtClose(KeyHandle2);
-        if (NT_SUCCESS(Status))
+        if (NT_SUCCESS(Status) &&
+            (PartialInfo->Type == REG_DWORD) &&
```

Comment:
Type and Datalength to be added to the DPRINT1 upon failure ? (currently prints Status only)

Reply:
Type added, DataLength wanted still.